guidelines/terms/20/audio-description.html

```diff
@@ -9,8 +9,8 @@
       other visual content.
    </p>
    
-   <p class="note">In standard audio description, narration is added during existing pauses in dialogue.
-      (See also <a>extended audio description</a>.)
+   <p class="note">In standard audio description, narration is added during existing pauses in dialogue. Usually, this may require
```

Comment:
I think "usually" is an overreach. Many instructional videos do no have music or non-informational sound.

```suggestion
   <p class="note">In standard audio description, narration is added during existing pauses in dialogue. This may require
```